src/System.Management.Automation/engine/Utils.cs

```diff
@@ -2078,6 +2122,13 @@ public static void SetTestHook(string property, object value)
             }
         }
 
+        /// <summary>This member is used for internal test purposes.</summary>
+        public static object GetTestHookValue(string property)
```

Comment:
Fields and properties aren't the same, so best to be specific.

```suggestion
        public static object GetTestHookValue(string field)
```